docs/rst/user_manual/configuration.rst

```diff
@@ -333,6 +333,40 @@ By setting the ``remove-unused-entities`` option to ``true``, the internal entit
 .. warning::
   At the time being, the removal of unused entities is incompatible with the `Transient-Local Durability QoS <https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/core/policy/standardQosPolicies.html#durabilityqospolicy>`_.
 
+.. _user_manual_configuration_dynamic_trigger:
+
+Discovery Trigger
+-----------------
+
+``specs`` supports a ``discovery-trigger`` **optional** value that configures what type of external entity triggers the discovery callbacks in the |ddsrouter|.
```

Comment:
Maybe this is too low level for a user, maybe it would be better just to say "triggers entities creation/removal" instead of "triggers the discovery callbacks".

Reply:
Update also the table.